keep/actions/actions_factory.py

```diff
@@ -0,0 +1,82 @@
+import time
+import logging
+import yaml
+from io import StringIO
+from uuid import uuid4
+from typing import List, Union
+from keep.api.models.action import ActionDTO
+from keep.api.models.db.action import Action
+from keep.api.core.db import get_all_actions, create_actions, delete_action, get_action, update_action
+from keep.actions.actions_exception import ActionsFactoryException
+
+logger = logging.getLogger(__name__)
+
+class ActionsFactory:
```

Comment:
Why did you chose to name it Factory although its just a CRUD?

Reply:
Yeah, think you're right on changing this to CRUD. At the beginning I thought there should be more logic to be put in this module. It's CRUD by now so I just changed to `ActionsCRUD`